src/include/mutex.h

```diff
@@ -65,16 +66,23 @@ struct __wt_rwlock {			/* Read/write lock */
  * Implemented as a macro so we can pass in a statistics field and convert
  * it into a statistics structure array offset.
  */
-#define	WT_RWLOCK_INIT_TRACKED(session, l, name) do {                   \
-	WT_RET(__wt_rwlock_init(session, l));                           \
-	(l)->stat_read_count_off = (int16_t)WT_STATS_FIELD_TO_OFFSET(   \
-	    S2C(session)->stats, lock_##name##_read_count);             \
-	(l)->stat_write_count_off = (int16_t)WT_STATS_FIELD_TO_OFFSET(  \
-	    S2C(session)->stats, lock_##name##_write_count);            \
-	(l)->stat_app_usecs_off = (int16_t)WT_STATS_FIELD_TO_OFFSET(    \
-	    S2C(session)->stats, lock_##name##_wait_application);       \
-	(l)->stat_int_usecs_off = (int16_t)WT_STATS_FIELD_TO_OFFSET(    \
-	    S2C(session)->stats, lock_##name##_wait_internal);          \
+#define	WT_RWLOCK_INIT_TRACKED(session, l, name) do {			\
+	WT_RET(__wt_rwlock_init(session, l));				\
+	(l)->stat_read_count_off = (int16_t)WT_STATS_FIELD_TO_OFFSET(	\
+	    S2C(session)->stats, lock_##name##_read_count);		\
+	(l)->stat_write_count_off = (int16_t)WT_STATS_FIELD_TO_OFFSET(	\
+	    S2C(session)->stats, lock_##name##_write_count);		\
+	(l)->stat_app_usecs_off = (int16_t)WT_STATS_FIELD_TO_OFFSET(	\
+	    S2C(session)->stats, lock_##name##_wait_application);	\
+	(l)->stat_int_usecs_off = (int16_t)WT_STATS_FIELD_TO_OFFSET(	\
+	    S2C(session)->stats, lock_##name##_wait_internal);		\
+} while (0)
```

Comment:
I don't understand what changed in these lines - was there a different spacing?

Reply:
It replaced spaces with tabs.